zipline/utils/calendars.py

```diff
@@ -0,0 +1,11 @@
+# flake8: noqa
```

Comment:
Like you I don't know of a use case off the top of my head where we'd need this; presumably if the calendars are different, we'd want to register them differently. Do you think it's still worth adding the `multipledispatch` logic in case we need it later?

Reply:
I think we can probably punt on this until we have a use-case for it.